### PR TITLE
Renovate/aws sdk js v3 monorepo

### PR DESCRIPTION
**Pull request by aochsner**
## Hey, I just made a Pull Request!

Fix https://github.com/backstage/backstage/pull/20461

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [x] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by aochsner** [comment]
@freben  i'm a bit confused on these test errors.  They don't seem related (database related for the most part) and they pass on my local... thoughts?

**Comment by freben** [comment]
Yeah don't worry, that's a frustrating flakiness in the tests.

**Comment by aochsner** [comment]
Good to go?

File: .changeset/clever-houses-scream.md [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+'@backstage/plugin-fossa': patch
+'@backstage/plugin-ilert': patch
+'@backstage/plugin-kafka': patch
+'@backstage/plugin-nomad': patch
+'@backstage/plugin-vault': patch
+'@backstage/plugin-gocd': patch
+'@backstage/plugin-home': patch
+'@backstage/plugin-todo': patch
+'@backstage/plugin-adr': patch
+'@backstage/plugin-org': patch
```
Comment (freben) [inline .changeset/clever-houses-scream.md:232]: What happened here? 😅 Should only have techdocs-node in here

**Comment by aochsner** [comment]
thanks!  missed that bit not sure why all that got generated

**Pull request merged**